.github/workflows/ci-tests-pytorch.yml

```diff
@@ -61,6 +61,10 @@ jobs:
           - {os: "macOS-11", pkg-name: "pytorch", python-version: "3.8", pytorch-version: "1.13"}
           - {os: "ubuntu-20.04", pkg-name: "pytorch", python-version: "3.8", pytorch-version: "1.13"}
           - {os: "windows-2022", pkg-name: "pytorch", python-version: "3.8", pytorch-version: "1.13"}
+          # pytorch 2.0 Tests (currently pre-release, change this once it's released)
+          - {os: "macOS-11", pkg-name: "lightning", python-version: "3.9", pytorch-version: "2.0", release: "pre"}
+          - {os: "ubuntu-20.04", pkg-name: "lightning", python-version: "3.9", pytorch-version: "2.0", release: "pre"}
+          - {os: "windows-2022", pkg-name: "lightning", python-version: "3.9", pytorch-version: "2.0", release: "pre"}
```

Comment:
These should be added to the checkgroup config

Reply:
shouldn't we wait with this until they have an official rc?

Reply:
It's out now

Reply:
yes, as @Borda shared on slack, we would merge it without adding it here, to first make sure they pass and once we're confident it works as it should, we would add it to checkgroup.
This PR only adds CI but does not aim at fixing issues.

Reply:
Please don't merge this with failing jobs, even if they are unmarked as required. CI should always be green.

I'm fine with fixing the issues separately, but then I would fix those first. You can work here to validate the changes and pull them to a separate PR once they are passing

Reply:
> Please don't merge this with failing jobs, even if they are unmarked as required. CI should always be green.

ls help us to make it green :)